requirements.txt

```diff
@@ -1,3 +1,12 @@
-datajoint>=0.13.0
-element-interface>=0.4.0
-plotly
+datajoint==0.14.0
+element_interface==0.5.0
+ipywidgets==8.0.4
+matplotlib==3.7.0
+mkdocs_gen_files==0.4.0
+nd2==0.5.3
+numpy==1.23.5
+plotly==5.13.1
+sbxreader==0.2.2
+scipy==1.10.1
+setuptools==65.6.3
+suite2p==0.10.4.dev154+g39fdb21
```

Comment:
```suggestion
```

Reply:
We need to document this better, but we have settled on installing the calcium imaging acquisition and analysis packages in `workflow-calcium-imaging` (see [requirements.txt](https://github.com/datajoint/workflow-calcium-imaging/blob/1b8810652770e7d0505e1d1f1803fc85beb36d02/requirements.txt#L9-L12)) to allow a user to select the specific packages to install so that `element-calcium-imaging` does not install all packages.

Reply:
Oh okay, That make sense that way we shorten the installation time and the space. I'll check if there are other packages from element-calcium-imaging that need to be installed.